content/cli/usages.md

```diff
@@ -156,6 +156,10 @@ $ nest add <name> [options]
 
 Updates `@nestjs` dependencies in the `package.json` `"dependencies"` list to their `@latest` version.
 
+```bash
+$ nest update [options]
+```
+
```

Comment:
```suggestion
```

As we plan to deprecate this one, I'd say we don't need adding an example at this point